test/js/reducers/questionnaire.spec.js

```diff
@@ -350,21 +350,17 @@ describe('questionnaire reducer', () => {
 
     const step = find(resultState.data.steps, s => s.id === '17141bea-a81c-4227-bdda-f5f69188b0e7')
     expect(step.choices.length).toEqual(2)
-    expect(step.choices[1]).toEqual({
-      value: 'Maybe',
-      responses: {
-        'en': {
-          sms: [
-            'M',
-            'MB',
-            '3'
-          ],
-          ivr: [
-            'May'
-          ]
-        }
-      },
-      skipLogic: 'end'
+    expect(step.choices[1].value).toEqual('Maybe')
```

Comment:
I changed this so that changes in the dummy questionnaire at the bottom of this file don't affect it so easily. An example of a change that would break this test is adding language prompts to the dummy object.